deploy/build/README.Linux.md

```diff
@@ -93,7 +93,7 @@ The following steps work on CentOS 7. For other CentOS versions, these steps mig
 
     ```shell
     # You need to enter your password to run.
-    sudo ~/cloudberrydb/deploy/build/README.Ubuntu.bash
```

Comment:
Are the changes here necessary?

Reply:
Yes, of course. At the beginning of the doc, we clone the new repo `apache/cloudberry` to the local machine, so the directory name has changed from `cloudberrydb` to `cloudberry`.